### DISCUSSION TITLE
Showing the graph only for certain pipelines

### DISCUSSION THREAD

**Discussion by BrachyKnight** [DerYeger/yeger#161] [Q&A] [answered]
Hi!
I have seen your graph package and so far is really awesome! It's really useful since one of the missing features of turborepo is an interactive graph like the one in Nx.
The only thing that I think is missing is specifying for which pipelines the graph is launched. I know there is a filter in the topbar, but not from the cli. This is important since right now I have some persistent tasks that exceed the default value of 10. Since this is not configurable from turbo.json or your package, when I launch the graph it fails.
Is there a workaround?

**Comment by DerYeger** [comment]
Hi, just FYI, I haven't really worked on the turbo graph for quite some time, because the API provided by turbo is basically unusable data and I have to invoke the turbo CLI to get the graph data. I could look into adding an ENV variable for passing the packages.

However, I'd first like to understand why this is required, since you're interacting with the graph anyway (I assume).
Which `default value of 10` are you referring to?

**Reply by BrachyKnight** [reply]
It's throwing an error since it says that the concurrency is set for a maximum of 10 persistent tasks. Since I usually run the commands  with --concurrency 11 this isn't an issue. But the graph doesn't have an option to set that number and I can't configure it from turbo.json

**Reply by DerYeger** [reply, marked as answer]
I see. Please try v0.4.7, it should now invoke turbo with `--concurrency=100%`.

**Reply by BrachyKnight** [reply]
Thank you! You have been super quick!

**Reply by DerYeger** [reply]
Thank you for reporting and testing this issue!
